pkg/api/errors/errors.go

```diff
@@ -413,6 +413,16 @@ func IsInternalError(err error) bool {
 	return reasonForError(err) == unversioned.StatusReasonInternalError
 }
 
+// IsTooManyRequests determines if err is an error which indicates that there are too many requests
+// that the server cannot handle.
+func IsTooManyRequests(err error) bool {
```

Comment:
Maybe I missed something, why can't this be written in the same way as the functions above?